__app__/sitemapparser/sites/sumosear_ch.py

```diff
@@ -0,0 +1,27 @@
+from urllib.parse import urljoin
+from bs4 import BeautifulSoup
+
+#https://sumosear.ch/images/tags/carbondale-il/escorts
+#https://sumosear.ch/images/tags/carbondale-il/female-escorts
+
+tags = [
```

Comment:
```suggestion
categories = [
```
Let's call these categories or sections. Unfortunately in other crawlers we use both already :)